RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -111,9 +111,15 @@ class PumpOpsSynchronous {
     }
     
     internal func getPumpModel() throws -> PumpModel {
-        guard let pumpModel = try (pump.pumpModel ?? PumpModel(rawValue: getPumpModelNumber())) else {
+        if let pumpModel = pump.pumpModel {
+            return pumpModel
+        }
+
```

Comment:
Was wondering about what happens when a user changes the pump id in settings.  Should probably make sure that we get a new pumpstate.